osu.Game/Audio/Effects/Filter.cs

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using ManagedBass.Fx;
+using osu.Framework.Audio.Mixing;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+
+namespace osu.Game.Audio.Effects
+{
+    public class Filter : Component
+    {
+        public BQFType FilterType = BQFType.LowPass;
```

Comment:
public settable but doesn't update when set. move to ctor or make it actually update (probably ctor is a better fit).